src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public void draw() {
+        super.draw();
```

Comment:
Ensure that the `super.draw()` call is necessary. This depends on the implementation of the `Figure` class. If `Figure` does not have a `draw()` method or if it does not need to be called, this line can be removed.